### PR TITLE
Error Handling completed for Login page. Displays error message for I…

### PR DESCRIPTION
**Pull request by ameyutturkar**
…nvalid user

### CODE REVIEW

File: release/src/main/java/com/blogit/controllers/LoginController.java [hunk 1]
```diff
@@ -37,8 +37,9 @@ public String loginAction(@ModelAttribute LoginEntity loginEntity, Model model)
             return "userDashboard";
         }
         // TODO: this solution is time being. Will research for Spring boot error handling
-        return "errorPage";
+        //return "errorPage";
```
Comment (aapeshave) [inline release/src/main/java/com/blogit/controllers/LoginController.java]: remove unnecessary comments

File: release/src/main/resources/templates/login.html [hunk 2]
```diff
@@ -52,9 +52,10 @@ <h2 class="form-signin-heading">Please sign in</h2>
                 </label>
             </div>
 
-            <button class="btn btn-lg btn-primary btn-block" type="submit">
+            <button class="btn btn-lg btn-primary btn-block" type="submit" >
```
Comment (aapeshave) [inline release/src/main/resources/templates/login.html]: move error messages below sign up button

File: release/src/main/java/com/blogit/controllers/LoginController.java (hunk 1 above)
Comment (ameyutturkar) [inline release/src/main/java/com/blogit/controllers/LoginController.java]: Removed.

File: release/src/main/resources/templates/login.html (hunk 2 above)
Comment (ameyutturkar) [inline release/src/main/resources/templates/login.html]: Done

**Pull request merged**